core/sis-utility/src/main/java/org/apache/sis/measure/Units.java

```diff
@@ -1111,6 +1111,16 @@
      */
     public static final Unit<Dimensionless> PIXEL;
 
+    /**
```

Comment:
Those fields can be moved below `KILOMETRES_PER_HOUR` for keeping related units together.